src/responder/common/responder_common.c

```diff
@@ -970,6 +1054,47 @@ int sss_process_init(TALLOC_CTX *mem_ctx,
         rctx->client_idle_timeout = 10;
     }
 
+    if (rctx->socket_activated) {
+        ret = confdb_get_int(rctx->cdb, rctx->confdb_service_path,
+                             CONFDB_RESPONDER_IDLE_TIMEOUT,
+                             CONFDB_RESPONDER_IDLE_DEFAULT_TIMEOUT,
+                             &rctx->idle_timeout);
+        if (ret != EOK) {
+            DEBUG(SSSDBG_OP_FAILURE,
+                  "Cannot get the responder idle timeout [%d]: %s\n",
+                   ret, strerror(ret));
```

Comment:
While you're already changing the patch, please consider using `sss_strerror()` in new code.